server/src/user/model.rs

```diff
@@ -65,3 +65,114 @@ impl User {
         Ok(count)
     }
 }
+
```

Comment:
I'm strongly for moving the tests to another file

Reply:
Actually, the book states:

>  You’ll put unit tests in the src directory in each file with the code that they’re testing. The convention is to create a module named tests in each file to contain the test functions and to annotate the module with cfg(test).
> https://doc.rust-lang.org/book/ch11-03-test-organization.html#unit-tests

Reply:
Integration tests will be placed into a `tests` directory